dbsnp-importer/src/test/java/uk/ac/ebi/eva/dbsnpimporter/io/readers/SubSnpCoreFieldsReaderTest.java

```diff
@@ -139,20 +158,18 @@ public void testQuery() throws Exception {
 
     @Test
     public void testQueryWithDifferentAssembly() throws Exception {
-        SubSnpCoreFieldsReader fieldsReader = buildReader("Bos_taurus_UMD_3.1.1",
-                                                          Collections.singletonList("Primary_Assembly"), 2000);
-        List<SubSnpCoreFields> list = readAll(fieldsReader);
-        assertEquals(2, list.size());
-        for (SubSnpCoreFields subSnpCoreFields : list) {
-            assertNotNull(subSnpCoreFields);
-            assertTrue(expectedRsIds.containsKey(subSnpCoreFields.getRsId()));
-        }
+        reader = buildReader(CHICKEN_ASSEMBLY_4, Collections.singletonList(PRIMARY_ASSEMBLY), PAGE_SIZE);
+        List<SubSnpCoreFields> list = readAll(reader);
+
+        assertEquals(1, list.size());
+        assertEquals(snpInDifferentAssembly, list.get(0));
```

Comment:
Since this is testing an "edge case", maybe the variant could be initialized in here instead of making it an attribute? That would also make the description in the `setUp` method unnecessary.

Reply:
Done in 		d84bca9